### DISCUSSION TITLE
Лайки советам | практикам

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/content#2589] [Идеи]
У нас есть советы, они же практики, они же мнения | комментарии из блока «В работе». 

Мы хотим как-то выделять более ценные? Например «большим пальцем вверх» 👍

Update: Ещё хорошо бы эти лайки считать, чтобы мы могли когда-нибудь использовать эту популярность совета.

**Comment by solarrust** [comment]
Я бы использовала что-то вроде рейтинга, как вот тут, в обсуждениях, стрелочка вверх и счётчик. И точно не делала бы дизлайки

**Reply by furtivite** [reply]
Да, дизлайки точно мимо. А с рейтингом, мне нравится идея, как тут. Вот только надо будет объяснять дополнительно, как этим пользоваться. Кажется, даже сейчас не все понимают, как эти стрелочки работают.

**Reply by solarrust** [reply]
Можно дописать что-то вроде «топ» =)

**Comment by pepelsbey** [comment]
Лучше всего было бы собирать это где-то на нашей стороне и потом интегрировать в мету самого совета раз в сутки или раз в неделю, скажем. Ну либо хранить где-то ещё и использовать при сборке.

Один из вариантов — сортировать советы к статье по этому рейтингу. Примерно как order в CSS: без рейтинга советы идут в порядке появления, с рейтингом — опережают порядок и идут по убыванию.

**Reply by nlopin** [reply]
Мне нравится этот вариант, он звучит просто. С другой стороны, непонятно, как объяснить пользователю, что его голос учтётся только через день.

Теоретически можно досортировывать джаваскриптом уже после рендеринга, запросив свежие данные с сервера.

**Reply by igsekor** [reply]
Вдруг кто-то в пиар зальёт мету неправильную специально или случайно? Вряд ли мы просмотрим, но вероятность этого всё же сохраняется.

**Reply by nlopin** [reply]
100% в мете не стоит это хранить, это серверное апи

**Comment by igsekor** [comment]
Может быть всё таки отдадим сбор лайков (голосов / рейтинга) на сторону бэкенда? Можно даже грузить текущие лайки на этапе сборки, чтобы можно было сортировать советы, и не переносить всё на клиентскую часть. Обновления будут тем актуальнее, чем чаще мы пушим в `main`. Ну а установку лайков можно отдать уже работающей системе с отзывами для статей.
